kotlin/workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/AndroidXViewBinding.kt

```diff
@@ -0,0 +1,33 @@
+package com.squareup.workflow.ui
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+
+typealias ViewBindingInflater<BindingT> = (LayoutInflater, ViewGroup?, Boolean) -> BindingT
+
+/**
+ * Creates a [ViewBinding] that [inflates][bindingInflater] a [BindingT] to show renderings of type
+ * [RenderingT], using [showRendering].
+ *
+ * ```
+ * val HelloBinding: ViewBinding<Rendering> =
+ *   bindViewBinding(HelloGoodbyeLayoutBinding::inflate) { rendering, containerHints ->
+ *     helloMessage.text = rendering.message
+ *     helloMessage.setOnClickListener { rendering.onClick(Unit) }
+ *   }
+ * ```
+ *
+ * If you need to initialize your view before [showRendering] is called, create a [LayoutRunner]
+ * and create a binding using `LayoutRunner.bind` instead.
```

Comment:
This was intentional, because this links to the wrong `bind` method and the link adds more confusion than anything. There's a super old dokka issue about this: https://github.com/Kotlin/dokka/issues/80